keyboards/kiwikey/wanderland/keymaps/stanrc85/rules.mk

```diff
@@ -0,0 +1,2 @@
+USER_NAME := stanrc85
```

Comment:
This shouldn't be needed here, in theory, since it matches the keymap name. 
```suggestion
```

Reply:
Removed, thanks